### ISSUE TITLE
label validationHarvestingNoHLT does not exist

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#29397]
Since CMSSW_11_1_X_2020-04-02-1100, the workflows 140.0, 145.0 and 150.0 have been failing due to
```
HARVESTING:validationHarvestingNoHLT+dqmHarvestingFakeHLT
entry file:step3_inDQM.root
Step: HARVESTING Spec: ['validationHarvestingNoHLT', 'dqmHarvestingFakeHLT']
validationHarvestingNoHLT is not a possible harvesting type. Available are [
<many lines cut>
]
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02622/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_1_X_2020-04-02-1100/bin/slc7_amd64_gcc820/cmsDriver.py", line 56, in &lt;module&gt;
    run()
  File "/cvmfs/cms-ib.cern.ch/nweek-02622/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_1_X_2020-04-02-1100/bin/slc7_amd64_gcc820/cmsDriver.py", line 28, in run
    configBuilder.prepare()
  File "/cvmfs/cms-ib.cern.ch/nweek-02622/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_1_X_2020-04-02-1100/python/Configuration/Applications/ConfigBuilder.py", line 2127, in prepare
    self.addStandardSequences()
  File "/cvmfs/cms-ib.cern.ch/nweek-02622/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_1_X_2020-04-02-1100/python/Configuration/Applications/ConfigBuilder.py", line 759, in addStandardSequences
    getattr(self,"prepare_"+stepName)(sequence = '+'.join(stepSpec))
  File "/cvmfs/cms-ib.cern.ch/nweek-02622/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_1_X_2020-04-02-1100/python/Configuration/Applications/ConfigBuilder.py", line 2004, in prepare_HARVESTING
    getattr(self.process, name)
AttributeError: 'Process' object has no attribute 'validationHarvestingNoHLT'
```

**Comment by silviodonato** [comment]
fix in https://github.com/cms-sw/cmssw/pull/29393
I think I will integrate this in the next IB

**Comment by silviodonato** [comment]
solved by #29393
